src/graph/rg_matrix/rg_matrix.h

```diff
@@ -304,3 +304,10 @@ void RG_Matrix_free
 	RG_Matrix *C
 );
 
+// get matrix C without writing to internal matrix
+void RG_Matrix_get
```

Comment:
I think we need to find a better name for this method, also please add test for it

Reply:
Maybe RG_Matrix_export?